Applications/SlicerApp/Testing/Python/DICOMReaders.py

```diff
@@ -97,23 +97,27 @@ def test_Part1DICOM(self):
         "fileName": "Mouse-MR-example-where-GDCM_fails.zip",
         "name": "Mouse-MR-example-where-GDCM_fails",
         "seriesUID": "1.3.6.1.4.1.9590.100.1.2.366426457713813178933224342280246227461",
-        "expectedFailures": ["GDCM", "Archetype"]
+        "expectedFailures": ["GDCM", "Archetype"],
+        "voxelValueQuantity": "(DCM, 110852, MR signal intensity)",
+        "voxelValueUnit": "(UCUM, [hnsf'U], Hounsfield unit)"
```

Comment:
I did a quick search, and it looks like "units of measurement" and "unit of measurement" are used interchangeably. 

Either way is fine with me, I just think that perhaps in this case it is better to be consistent with the standard to reduce confusion and the mental effort needed by the users typing these things while correlating Slicer implementation with DICOM.